dist/tools/compile_test/compile_test.py

```diff
@@ -86,12 +89,33 @@ def get_results_and_output_from(fd):
             output.write(line)
 
 
+def get_app_dirs():
+    yield from check_output(["make", "-f", "makefiles/app_dirs.inc.mk", "info-applications"]) \
```

Comment:
The script is currently supposed to be python2 compatible and `python2` is used by default on ubuntu because of the shebang. `yield from` breaks this compatibility.

Also here, as `split` returns a list and not a generator, `yield from` adds nothing.
Even `check_output` and `decode` are not generator.

Reply:
ok, it is just returning now.